converters/fromZigbee.js

```diff
@@ -1506,6 +1506,25 @@ const converters = {
     // #endregion
 
     // #region Non-generic converters
+    ikuu_dimmer: {
+        cluster: 'manuSpecificTuya',
+        type: ['commandGetData', 'commandSetDataResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const dp = msg.data.dp; 
+            const value = tuya.getDataValue(msg.data.datatype, msg.data.data); 
+            switch (dp) {
+            case tuya.dataPoints.state:
+                    return {state: value ? 'ON': 'OFF'};
+            case tuya.dataPoints.ikuuDimmerLevel: 
+                    return {brightness: mapNumberRange(value, 10, 1000, 0, 254), level: value};
```

Comment:
Could you remove the `level` value here? (since it has no added value since `brightness` is already there)

Reply:
I believe I have done this. Sorry, I am still learning basically everything here.

I originally copied this line from the tuya dimmer section:
`return {brightness: mapNumberRange(value, 10, 1000, 0, 254), level: value};`

I wasn't really sure why it was there for similar devices.